iulianIachimov/src/main/java/dataCollectionTask/Models/Item.java

```diff
@@ -1,19 +1,50 @@
 package dataCollectionTask.Models;
 
 public class Item {
-  public long id;
-  public String title;
-  public Type type;
+  private final long id;
+  private String title;
+  private Type type;
+  private double price;
   private static long idCounter;
 
-  public Item(String title, Type type) {
+  public Item(String title, Type type, double price) {
     this.id = ++idCounter;
     this.title = title;
     this.type = type;
+    this.price = price;
   }
 
   @Override
   public String toString() {
     return "Item{" + "id=" + id + ", title='" + title + '\'' + ", type=" + type + '}';
   }
+
+  public long getId() {
+    return id;
+  }
+
+  public String getTitle() {
+    return title;
+  }
+
+  public void setTitle(String title) {
+
+    this.title = title;
+  }
+
+  public Type getType() {
+    return type;
+  }
+
+  public void setType(Type type) {
+    this.type = type;
+  }
+
+  public double getPrice() {
+    return price;
+  }
+
+  public void setPrice(double price) {
```

Comment:
encapsulation is used in order to get the totla control over the variables, can we have a negative price? Could you please handle this situation within the setPrice() method?

Reply:
Done

Reply:
The price should be positive always.